assets/src/features/leaving-page/utils/sendForm.tsx

```diff
@@ -13,9 +13,7 @@ const sendForm = async (form: Inputs) => {
   // Since the request is opaque, we have no direct information
   // if the response has completed successfully or not.
   // In case of wrong input values, the `fetch` function logs `REQ_ERR` on stderr.
-  fetch(url, { mode: "no-cors" }).then((response) => {
-    console.log("Response", response);
-  });
+  await fetch(url, { mode: "no-cors" });
```

Comment:
After a bit of deeper analysis, I think the proper way to do this is to proxy the request via backend. However, this is out of the scope of this PR. Can you create an issue for this as well?

Reply:
[Here it is](https://membraneframework.atlassian.net/jira/software/c/projects/MV/boards/8/backlog?view=detail&selectedIssue=MV-476&issueLimit=100)